src/java/picard/sam/markduplicates/EstimateLibraryComplexity.java

```diff
@@ -82,25 +82,39 @@
-                "MAX_DIFF_RATE (0.03 by default).\n\n" +
-                "Reads of poor quality are filtered out so as to provide a more accurate estimate. The filtering " +
-                "removes reads with any no-calls in the first N bases or with a mean base quality lower than " +
-                "MIN_MEAN_QUALITY across either the first or second read.\n\n" +
-                "Unpaired reads are ignored in this computation.\n\n" +
-                "The algorithm attempts to detect optical duplicates separately from PCR duplicates and excludes " +
-                "these in the calculation of library size. Also, since there is no alignment to screen out technical " +
-                "reads one further filter is applied on the data.  After examining all reads a Histogram is built of " +
-                "[#reads in duplicate set -> #of duplicate sets] all bins that contain exactly one duplicate set are " +
-                "then removed from the Histogram as outliers before library size is estimated.",
-        usageShort = "Estimates library complexity from the sequence of read pairs",
+        usage = EstimateLibraryComplexity.USAGE_SUMMARY + EstimateLibraryComplexity.USAGE_DETAILS,
+        usageShort = EstimateLibraryComplexity.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class EstimateLibraryComplexity extends AbstractOpticalDuplicateFinderCommandLineProgram {
-
+    static final String USAGE_SUMMARY = "Tool for estimating library complexity.   ";
+    static final String USAGE_DETAILS = "This tool is a quality metric for a sequencing library preparation." +
+            "<br /><br />Library complexity refers to the number of unique DNA fragments present in a given library.  " +
+            "Reductions in complexity compromise downstream analyses and can result from either PCR or optical duplicates (replicates)." +
+            "" +
+            "<br /><br />PCR-induced replicates can result from: inadequate amounts of starting material (genomic DNA, cDNA, etc.), " +
+            "losses during cleanups and size selection, and amplification-associated artifacts.   These replicates " +
+            "are removed via the creation of a histogram consisting of the numbers of reads in duplicate set vs. the numbers " +
+            "of duplicate sets.  All bins that contain exactly one duplicate set are then removed from the histogram prior" +
+            " to estimating library size.<br /> <br />" +
+            "" +
+            "Optical replicates result from reads that have identical sequences and whose clusters are proximal to each other.  " +
+            "These are identified using both the READ_NAME_REGEX string and the values for the OPTICAL_DUPLICATE_PIXEL_DISTANCE.<br /> <br />" +
+            "" +
+            "Sequence identity is determined by scanning the first five (default value) bases.  Sequence identity is indicated if" +
```

Comment:
I think this: "Sequence identity is indicated if" should be rephrased as: "Sequences are considered identical if"